src/main/java/algorithmic_task/AlgorithmicTaskApplication.java

```diff
@@ -0,0 +1,16 @@
+package algorithmic_task;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@EnableAutoConfiguration
```

Comment:
Please find out what annotation SpringBootApplication is doing and optimize this file